File: app/src/main/java/com/emarsys/catseeker/CatSeekerApp.java
package com.emarsys.catseeker;

import android.app.Application;
import android.content.Intent;

import com.emarsys.catseeker.network.ApiSecretProvider;
import com.emarsys.catseeker.service.CatSeekerService;

public class CatSeekerApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        ApiSecretProvider.init(this);
        startService(new Intent(this, CatSeekerService.class));
    }
}

File: app/src/main/kotlin/com/emarsys/catseeker/CatSeekerApp.kt
package com.emarsys.catseeker

import android.app.Application
import android.content.Intent
import com.emarsys.catseeker.model.MetaData
import com.emarsys.catseeker.service.CatSeekerService

class CatSeekerApp : Application() {

    override fun onCreate() {
        super.onCreate()
        MetaData.init(this)
        startService(Intent(this, CatSeekerService::class.java))
    }
}